parser/src/python.lalrpop

```diff
@@ -979,6 +980,25 @@ FuncDef: ast::Stmt = {
     },
 };
 
+TypeAliasName: ast::Expr = {
+    <location:@L> <name:Identifier> <end_location:@R> => ast::Expr::Name(
+        ast::ExprName { id: name, ctx: ast::ExprContext::Load, range: (location..end_location).into() },
+    ),
+}
```

Comment:
Nit: You may want to factor the `Expr::Name` parsing into its own named rule and re-use for `TypeAliasName`, `MatchName`, and `Atom<Goal`>`

Reply:
I definitely stole the `MatchName` implementation for this although I did not notice it overlaps with `Atom<Goal>`. I'd rather tackle this as a follow-up. I'll create an issue.

Reply:
#98